### DISCUSSION TITLE
How use MudSelect like custom component

### DISCUSSION THREAD

**Discussion by xlogex** [fullstackhero/blazor-wasm-boilerplate#186] [Q&A]
Hi i would like if it is able to customize like component 
<MudSelect MultiSelection="true" ... >
best regard

**Comment by xlogex** [comment]
https://try.mudblazor.com/snippet/ckQGuzkjKeQGnCTC

**Comment by MikaelHild** [comment]
Hi, 
Are you referring to the ColorPanel of this project, or do you want to create a component based on the MudSelect component from Mudblazor? 

https://github.com/fullstackhero/blazor-wasm-boilerplate/blob/main/src/Client/Components/ThemeManager/ColorPanel.razor

**Reply by xlogex** [reply]
i would like to create a component based on the MudSelect component

**Reply by neozhu** [reply]
> i would like to create a component based on the MudSelect component

could give me one sample
